### PR TITLE
kubernetes cluster setup with kube-router using kubeadm

### PR DESCRIPTION
**Pull request by murali-reddy**
Setup instruction to setup cluster with kube-router

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/5189)

### CODE REVIEW

File: docs/setup/independent/create-cluster-kubeadm.md [hunk 1]
```diff
@@ -249,6 +249,16 @@ kubectl apply -f https://raw.githubusercontent.com/coreos/flannel/master/Documen
 ```
 {% endcapture %}
 
+{% capture kube-router %}
+
+Kube-router relies on kube-controll-manager to allocate pod CIDR for the nodes. So you must use kubeadm init with --pod-network-cidr flag. 
+
+Kube-router provides pod networking, network policy and high perfoming IPVS/LVS based service proxy as well. 
+
+Please follow the instruction [here](https://github.com/cloudnativelabs/kube-router/blob/master/Documentation/kubeadm.md) to setup Kubernetes cluster with Kube-router using kubeadm.
```
Comment (radhikapc) [inline docs/setup/independent/create-cluster-kubeadm.md]: For information on setting up Kubernetes cluster with Kube-router using kubeadm, see [kubeadm documentation](https://github.com/cloudnativelabs/kube-router/blob/master/Documentation/kubeadm.md).

**Comment by radhikapc** [comment]
@murali-reddy :wave:

File: docs/setup/independent/create-cluster-kubeadm.md (hunk 1 above)
Comment (murali-reddy) [reply docs/setup/independent/create-cluster-kubeadm.md]: @radhikapc thanks. have incorporated changes.

File: docs/setup/independent/create-cluster-kubeadm.md [hunk 2]
```diff
@@ -249,6 +249,16 @@ kubectl apply -f https://raw.githubusercontent.com/coreos/flannel/master/Documen
 ```
 {% endcapture %}
 
+{% capture kube-router %}
+
+Kube-router relies on kube-controll-manager to allocate pod CIDR for the nodes. So you must use kubeadm init with --pod-network-cidr flag.
+
+Kube-router provides pod networking, network policy and high performing IPVS/LVS based service proxy as well.
```
Comment (radhikapc) [inline docs/setup/independent/create-cluster-kubeadm.md]: typo: performing
Kube-router provides pod networking, network policy, and high-performing IP Virtual Server(IPVS)/Linux Virtual Server(LVS) based service proxy.

Comment (radhikapc) [reply docs/setup/independent/create-cluster-kubeadm.md]: consistency: `kube-router` vs `Kube-router`

File: docs/setup/independent/create-cluster-kubeadm.md [hunk 3]
```diff
@@ -249,6 +249,16 @@ kubectl apply -f https://raw.githubusercontent.com/coreos/flannel/master/Documen
 ```
 {% endcapture %}
 
+{% capture kube-router %}
+
+Kube-router relies on kube-controll-manager to allocate pod CIDR for the nodes. So you must use kubeadm init with --pod-network-cidr flag.
```
Comment (radhikapc) [inline docs/setup/independent/create-cluster-kubeadm.md]: Kube-router relies on kube-controll-manager to allocate pod CIDR for the nodes. Therefore, use `kubeadm init` with the `--pod-network-cidr` flag.

File: docs/setup/independent/create-cluster-kubeadm.md [hunk 4]
```diff
@@ -272,8 +282,8 @@ kubectl apply -f "https://cloud.weave.works/k8s/net?k8s-version=$kubever"
 ```
 {% endcapture %}
 
-{% assign tab_names = "Choose one...,Calico,Canal,Flannel,Romana,Weave Net" | split: ',' | compact %}
-{% assign tab_contents = site.emptyArray | push: choose | push: calico | push: canal | push: flannel | push: romana | push: weave_net %}
+{% assign tab_names = "Choose one...,Calico,Canal,Flannel,Kube-router,Romana,Weave Net" | split: ',' | compact %}
+{% assign tab_contents = site.emptyArray | push: choose | push: calico | push: canal | push: flannel | push: kube-router | push: romana | push: weave_net %}
```
Comment (radhikapc) [inline docs/setup/independent/create-cluster-kubeadm.md]: kube-router vs Kube-router (consistent usage)

Comment (murali-reddy) [reply docs/setup/independent/create-cluster-kubeadm.md]: @radhikapc  I have made changes as suggested, except for one use of lower case kube-route, which is not part of displayed text, but its the template tabs internal name.

Comment (radhikapc) [reply docs/setup/independent/create-cluster-kubeadm.md]: Do you mean `kube-router` is used internally, whereas while referring to the service we use `Kube-router`?

**Comment by murali-reddy** [comment]
Yes it's part of markdown template stanza.

**Comment by radhikapc** [comment]
@bowei Over to you for a tech review.

**Pull request merged**